staking/src/weights.rs

```diff
@@ -42,6 +42,8 @@
 use frame_support::{traits::Get, weights::{Weight, constants::RocksDbWeight}};
 use sp_std::marker::PhantomData;
 
+const EXTRINSIC_FIXED_WEIGHT: Weight = 100_000_000;
```

Comment:
@ekovalev please check it. It should match 0.0007 XOR

Reply:
it's still subject to normalization via the polymomial WeightToFee transform. The coefficients of the transform are now configured in such a way that the result is 0.0007 XOR for 100_000_000 units of weight